lib/livebook_web/live/file_select_component.ex

```diff
@@ -181,14 +186,14 @@ defmodule LivebookWeb.FileSelectComponent do
         </div>
       </div>
 
-      <.new_item_section type="dir" icon="folder-add-fill" myself={@myself} />
-      <.new_item_section type="notebook" icon="file-add-line" myself={@myself} />
+      <.new_item_section id={@id} type="dir" icon="folder-add-fill" myself={@myself} />
+      <.new_item_section id={@id} type="notebook" icon="file-add-line" myself={@myself} />
```

Comment:
Same here, I would prefer to build the new ID here instead of inside the component. Because `id={@id}` will always make me pause and double check.